etc/picongpu/wexac-weizmann/gpu.tpl

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+# Copyright 2013-2021 Axel Huebl, Anton Helm, Richard Pausch, Rene Widera,
+#                     Marco Garten
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for hemera's SLURM batch system
```

Comment:
Hemera -> Wexac ? Also, below the queue is named k20, wondering if this is also a Hemera leftover

Reply:
Thanks for catching this - I thought I removed it 😓

Reply:
I forgot to save the while in VS Code before committing 😓  - should be fixed now.